documentation/metrics-api/v2/metrics_api.markdown

```diff
@@ -0,0 +1,182 @@
+---
+layout: default
+title: "Puppet Server: Metrics API v2"
+canonical: "/puppetserver/latest/metrics-api/v2/metrics_api.html"
+---
+
+By default, Puppet Server enables two optional web APIs for
+[Java Management Extension (JMX)](https://docs.oracle.com/javase/tutorial/jmx/index.html)
+metrics, namely
+[managed beans (MBeans)](https://docs.oracle.com/javase/tutorial/jmx/mbeans/). For the older metrics API, see [the `/metrics/v1` documentation](../v1/metrics_api.html).
+
+## Jolokia endpoints
+
+The v2 metrics endpoint uses the [Jolokia](https://jolokia.org) library, an extensive open-source metrics library with its own documentation.
+
+The documentation below provides only the information you need to use the metrics as configured by default for Puppet Server, but Jolokia offers more features than are described below. Consult the [Jolokia documentation](https://jolokia.org/documentation.html) for more information.
+
+For security reasons, we enable only the read-access Jolokia interface by default:
+
+-   `read`
+-   `list`
+-   `version`
+-   `search`
+
+### Configuring Jolokia
+
+To change the security access policy, create the `/etc/puppetlabs/puppetserver/jolokia-access.xml` file with contents that follow the [Jolokia access policy](https://jolokia.org/reference/html/security.html) and uncomment the `metrics.metrics-webservice.jolokia.servlet-init-params.policyLocation` parameter before restarting puppetserver.
+
+The `metrics.metrics-webservice.jolokia.servlet-init-params` table
+within the [`/etc/puppetlabs/puppetserver/conf.d/metrics.conf`](../../config_file_metrics.markdown) file provides more configuration options. See Jolokia's [agent initialization documentation](https://jolokia.org/reference/html/agents.html#agent-war-init-params) for all of the available options.
```

Comment:
@justinstoller Do you mean that the docs for this file don't exist yet or that the metrics.conf file itself doesn't exist?  I would have expected a file at `/etc/puppetlabs/puppetserver/conf.d/metrics.conf` which looks like [this](https://github.com/puppetlabs/puppetserver/blob/bd05e6ef2ca0229c82526928843085bd50f0e58f/ezbake/config/conf.d/metrics.conf) to be present after installation of a Puppet Server-5 based OSS package.  Are you seeing that?